auto_causality/params.py

```diff
@@ -460,22 +460,28 @@ def _configs(self):
                     "model_y_xw": outcome_model,
                     "model_t_xw": propensity_model,
                     "model_z_xw": deepcopy(propensity_model),
-                }
+                },
+                search_space={
+                    "fit_cate_intercept": tune.choice([0, 1]),
+                    "mc_agg": tune.choice(["mean", "median"]),
+                },
+                defaults={
+                    "fit_cate_intercept": 0,
+                    "mc_agg": "mean",
+                },
             ),
             "iv.econml.iv.dml.DMLIV": EstimatorConfig(
                 init_params={
                     "model_y_xw": outcome_model,
                     "model_t_xw": propensity_model,
-                    "model_t_xwz": deepcopy(propensity_model),
-                    "model_final": final_model,
-                    "discrete_treatment": False,
+                    # "model_t_xwz": deepcopy(propensity_model),
+                    # "model_final": final_model,
+                    "fit_cate_intercept": True,
```

Comment:
This should be removed, as will be (hopefully) overridden by the search version. Each parameter should only occur in init params OR in search_space + defaults